Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -171,6 +178,14 @@ def check_feed(client: Client) -> str:
 def main():
     params = demisto.params()
     server_url = (params.get('server_url')).rstrip()
+    default_headers = params.get('default_headers')
+    if default_headers:
```

Comment:
This will break BC since users of existing instances will not have the default value from the YML in their instance configuration. Please add the previous USER_AGENT when `default_headers` is missing.

Reply:
done